src/uu/mv/src/mv.rs

```diff
@@ -85,16 +82,8 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         .unwrap_or_default();
 
     let overwrite_mode = determine_overwrite_mode(&matches);
-    let backup_mode = backup_control::determine_backup_mode(
-        matches.is_present(OPT_BACKUP_NO_ARG),
-        matches.is_present(OPT_BACKUP),
-        matches.value_of(OPT_BACKUP),
-    );
-    let backup_mode = match backup_mode {
-        Err(err) => {
-            show_usage_error!("{}", err);
-            return 1;
-        }
+    let backup_mode = match backup_control::determine_backup_mode(&matches) {
+        Err(_) => return 1,
```

Comment:
I think you're suppressing the error message here (i.e. not printing the message). Is that intentional?

Reply:
Nope, that was an oversight. Thanks!